docs/content/how-to-guides/configure-reactions/configure-mcp-reaction/_index.md

```diff
@@ -0,0 +1,320 @@
+---
+type: "docs"
+title: "Configure an MCP Reaction"
+linkTitle: "Configure an MCP Reaction"
+weight: 90
+description: >
+    Learn how to configure an MCP Reaction
+---
+
+The MCP Reaction provides a Model Context Protocol (MCP) server that enables MCP clients to connect and subscribe to Drasi queries for real-time updates. MCP clients can use the resource subscription feature to receive notifications when query results change, making it ideal for integrating Drasi with AI agents and other MCP-compatible tools.
+
+## Requirements
+
+On the computer from where you will create the Reaction, you need the following software:
+- [Drasi CLI](/reference/command-line-interface/)
+
```

Comment:
nit: `kubectl` is used later on in the local dev section. Should we add it as an optional requirement